maintenance.py

```diff
@@ -81,25 +104,26 @@ def wait_for_maintenance(callback):
             last_maintenance_event = maintenance_event
             callback(maintenance_event)
 
-
 def maintenance_callback(event):
     if event:
-        # print('Undergoing host maintenance: {}'.format(event))
+        logger.warning('Undergoing host maintenance: %s', event)
```

Comment:
Should be info rather than warning, yeah?

Reply:
this is as the customer asked - the thought process: if we're undergoing a maintenance event, this is "not normal" for the system (but is fine for the script) hence warning and not info.

Reply:
From the script's perspective, it should still be INFO right. The script is meant to mitigate the impact during "live migration" or "termination" and starts to execute the logic as soon as (60 secs actually before the maintenance event starts) the `maintenance-event` attribute is set (like a response to an event). So the maintenance is scheduled already.
I may be wrong. Let me know your thoughts.

Reply:
@spkesan you're right - but think of it from the customer perspective - it's either looking for a plain info saying "something bad is going on" (an event, meaning changes to his system) or to have it as bold "warning" stating the same.

the warning in that case is not that the script is doing something bad, is that the system is undergoing something bad...

Reply:
Okay. A WARNING is good in that case.